test/SparqlExpressionTest.cpp

```diff
@@ -720,7 +720,26 @@ TEST(SparqlExpression, customNumericFunctions) {
                       std::vector<Id>{D(0), D(tan(1)), D(tan(2)), D(tan(-1))});
 }
 
-// ________________________________________________________________________________________
+// ____________________________________________________________________________
+TEST(SparqlExpression, isSomethingFunctions) {
+  const auto T = Id::makeFromBool(true);
+  const auto F = Id::makeFromBool(false);
+  // TODO(hannahbast): Without the \"\"lit, the last test fails. Why?
```

Comment:
There is a lot of stuff that should be updated here once we have the proper handling of the Iris and literals, but I currently don't understand the question, but maybe that also isn't important for now.

Reply:
The question is why I need `\"\"` here (two quotes instead of one) for the test to work?

Reply:
Let us not worry too much about this, it will all go away rather sooner than later with the work of @greenBene .
(It tests cases that are currently broken anyway).